version.ml

```diff
@@ -0,0 +1,7 @@
+let version = 
+  try
+    let ic = Unix.open_process_in "git describe --tags --always --dirty 2>/dev/null" in
+    let version = input_line ic in
+    close_in ic;
```

Comment:
In general, one should use `Unix.close_process_in` instead of a plain `close_in` to close the channel returned by `Unix.open_process_in`, to avoid leaking the process handle. Not very important here because this is a short-lived process anyway, but just to be correct.

Reply:
Fixed, thanks!